### ISSUE TITLE
Playlist Enhancement

### ISSUE THREAD

**Issue by Natemo6348** [galnir/Master-Bot#76]
Maybe you could make it where if a video is private in a playlist it skips it instead of returning an error for the entire playlist. (Error: resource youtube#videoListResponse not found) I assume this is from discord.js but i'm not sure. Also maybe make the queue show in pages so that the embed doesn't exceed the maximum size of 6000. (DiscordAPIError: Invalid Form Body
embed: Embed size exceeds maximum size of 6000)

**Comment by galnir** [comment]
Commit 7ce91adb addresses the embed maximum size error. In my opinion, using the queue command after adding a large playlist will spam the chat, so now the bot will display the first 10 songs in the queue. I'll implement a sophisticated queue command in the future so "paging" will be available.
Commit 2151194b7 adresses the playlist not playing if one video is private issue.

**Comment by Natemo6348** [comment]
@galnir i get the maximum size error when shuffling as well, and when i play a playlist with a private video in it, it puts the other videos in the queue and won't play unless i add another song to the queue

**Comment by galnir** [comment]
@Natemo6348 Can you send me the playlist link?

**Comment by Natemo6348** [comment]
@galnir here's the playlist link https://www.youtube.com/playlist?list=PLckeMyCaCCIN_JU1V4oADW50DlGOREoLj

**Comment by galnir** [comment]
@Natemo6348 The error is emitted because one of the videos got deleted from YouTube, I attempted to fix that issue in 428bbb2(let me know if that doesn't work because for me it does).
Nevertheless don't try to play long playlists like the one you linked, because it will make your API key hit its daily limit. Processing 4k songs is not a good idea to say the least. I tried playing that playlist twice and my key hit its limit
